src/Strategy/RPM.hs

```diff
@@ -0,0 +1,132 @@
+module Strategy.RPM (
+  discover,
+  getSpecDeps,
+  getTypeFromLine,
+
+  RPMDependency(..),
+  RequiresType(..),
+  Dependencies(..)
+) where
+
+import Prologue
+
+import Control.Effect.Error
+import qualified Data.Map.Strict as M
+import Data.Maybe (mapMaybe)
+import qualified Data.Text as T
+import DepTypes
+import Discovery.Walk
+import Effect.Grapher
+import Effect.ReadFS
+import Graphing (Graphing)
+import Types
+
+newtype SpecFileLabel =
+  RequiresType DepEnvironment
+  deriving (Eq, Ord, Show, Generic)
+  
+data RPMDependency = RPMDependency 
+  { rpmDepName :: Text,
+    rpmConstraint :: Maybe VerConstraint
+  } deriving (Eq, Ord, Show, Generic)
+
+{- These two structures should make it very easy to add other headers like `Requires`
```

Comment:
Sure, I can do that.

Reply:
Actually, this brings up an interesting point.  Should `BuildRequires` and `Requires` both be treated as production deps, or should `BuildRequires` be treated as a dev dependency?  Currently it's dev.

Reply:
(Chatted offline) We should not tag `BuildRequires` anything and omit `Requires` from the dependency graph until we have a need for including it. Right now we wanted to make sure that we also parse `Requires` to prove that it can be easily added in the future if needed.